### DISCUSSION TITLE
[DOCS&SAMPLES REQUEST]: Extension packages: Delta, Hyperspace, Synapse

### DISCUSSION THREAD

**Discussion by pherbel** [dotnet/spark#578] [Q&A]
Hello,

.Net Spark is great. We are happy to start experimenting it.

We are interested in using Delta format, Hyperspace index and Synapse Analytics. 
As I saw that there are extension packages for that. That's great.

Could you share some info on how we can use these tools?
I couldn't find any sample and install guide.

**Comment by suhsteve** [comment]
@pherbel There are [nugets available ](https://github.com/dotnet/spark/blob/master/src/csharp/Extensions/README.md) for the Delta and Hyperspace extensions.  Some examples of how to call the libraries can be found in the [Delta](https://github.com/dotnet/spark/tree/master/src/csharp/Extensions/Microsoft.Spark.Extensions.Delta.E2ETest) and [Hyperspace](https://github.com/dotnet/spark/tree/master/src/csharp/Extensions/Microsoft.Spark.Extensions.Hyperspace.E2ETest) tests.  In the coming weeks, these nugets will soon be installed as part of the default experience in Synapse Analytics.

**Comment by pherbel** [comment]
Yeah, I found the tests and nugets and I'm testing it.

https://github.com/dotnet/spark/issues/580#

Sounds good, but also I think it would be useful for devs if we have small docs for local development too.

- I have to download jar file for Maven repository and install it in local spark directory
- Could you provide jar files in the GitHub release and not just source code?
- Need C# sample

Thanks
